packages/app-landing/src/Navigate.tsx

```diff
@@ -0,0 +1,233 @@
+            </Link>
+          </div>
+        </div>
+        <div className='feature'>
+          <div className='icon'>
+            <FontAwesomeIcon icon={faExchangeAlt} color='blue' size='3x' />
+          </div>
+          <div className='header'>Send assets</div>
+          <div className='content'>
+            Send your CENNZnet assets between addresses.
+          </div>
+          <div className='action'>
+            <Link to='/generic-asset'>
+              <button className='button'>Go to Send assets</button>
+            </Link>
+          </div>
+        </div>
+        <div className='feature'>
+          <div className='icon'>
+            <FontAwesomeIcon icon={faAddressBook} color='blue' size='3x' />
+          </div>
+          <div className='header'>Create an address book</div>
+          <div className='content'>
+            Make sending transactions easier by creating an address book of the
+            accounts that you frequently interact with.
+          </div>
+          <div className='action'>
+            <Link to='/addressbook'>
+              <button className='button'>Go to Address book</button>
+            </Link>
+          </div>
+        </div>
+        <div className='feature'>
+          <div className='icon'>
+            <FontAwesomeIcon icon={faCompass} color='blue' size='3x' />
+          </div>
+          <div className='header'>Explore network activities</div>
+          <div className='content'>Find out what's happening on CENNZnet.</div>
+          <div className='action'>
+            <Link to='/explorer'>
+              <button className='button'>Go to Explorer</button>
+            </Link>
+          </div>
+        </div>
+        <div className='feature'>
+          <div className='icon'>
+            <FontAwesomeIcon icon={faTools} color='blue' size='3x' />
+          </div>
+          <div className='header'>Developer tools</div>
+          <div className='content'>
+            Looking for something more advanced? Find all the developer tools here.
+          </div>
+          <div className='action'>
+            <Link to='/contracts'>
+              <button className='button'>Open Advanced</button>
+            </Link>
+          </div>
+        </div>
+        <div className='feature-not-supported'>
+          <div className='icon'>
+            <FontAwesomeIcon icon={faCoins} color='black' size='3x' />
+          </div>
+          <div className='header'>Staking / Delegate</div>
+          <button className='no-action-button'>Coming soon</button>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export default styled(Navigate)`
+  padding-top: 90px;
+  height: 983px;
+  width: 100%;
+  background-color: #fafafa;
+
+  .explore-featured-fun {
+    margin-left: 79px;
+    height: 58px;
+    width: 477px;
+    color: #5c5d5d;
+    font-family: Lato;
+    font-size: 22px;
+    font-weight: bold;
+    letter-spacing: 0;
+    line-height: 32px;
+  }
+
+  .grid-container {
```

Comment:
`styled-components` make us be able to make html elements(react components) and css styles be united together as a distributed component without being bridged with classname. classname is fine, but we have to define css classname naming convention structurally (like using BEM styles) and carefully, otherwise if we defined `grid-container` classname styles somewhere else in the app, its hard to track and workaround.

In the future, we could refactor styles with classname to be `styled-components` way so that we do not need to worry about classnaming convention, for example here could be refactored like:

```js
const GridContainer = styled.div`
  ...
`;
// Then refactor code in component html element from <div classname="grid-container">...<div/>
<GridContainer>
  ...
<GridContainer/>
```